zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/job/CompleteJobTest.java

```diff
@@ -69,6 +72,8 @@ public void shouldCompleteJob() {
         .hasRetries(job.getRetries())
         .hasDeadline(job.getDeadline())
         .hasTenantId(TenantOwned.DEFAULT_TENANT_IDENTIFIER);
+
+    assertFalse(recordValue.getResult().isDenied());
```

Comment:
🔧 It seems uncommon to use JUnit 4 assertions (`org.junit.Assert.*`) in our tests. I’d recommend switching to AssertJ for consistency:
```java
    assertThat(recordValue.getResult().isDenied()).isFalse();
```
Also, you could use the chain `hasResult` method from the autogenerated assertions for `JobRecordValue`:
```java
    Assertions.assertThat(recordValue)
        .hasWorker(batchRecord.getValue().getWorker())
        .hasType(job.getType())
        .hasRetries(job.getRetries())
        .hasDeadline(job.getDeadline())
        .hasTenantId(TenantOwned.DEFAULT_TENANT_IDENTIFIER)
        .hasResult(new JobResult().setDenied(false));
```

Reply:
Great suggestion! Thanks @ce-dmelnych !